README.md

```diff
@@ -121,14 +121,17 @@ Next clone the CI script:
 
 Manually define the variables, Travis would otherwise define for you. These are required:
 
-    export TRAVIS_BRANCH=melodic-devel
+    export TRAVIS_BRANCH=melodic-devel   # Replace with the repository branch you intend to test against CI
```

Comment:
```suggestion
    export TRAVIS_BRANCH=$(git rev-parse --abbrev-ref HEAD)
```
What do you think about this?

Reply:
No, this doesn't refer to the current local branch, but to the branch a github PR should be merged into. I guess this can only be manually specified (and should actually read something like origin/melodic-devel). However, this is only required for clang-tidy's check for modified files.

Reply:
ok, than maybe that information should be in the comment
```suggestion
    export TRAVIS_BRANCH=melodic-devel   # the branch a github PR should be merged into, e.g. `origin/melodic-devel`. Only required for clang-tidy's check for modified files.
```

Reply:
Thank you for the suggestions